demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -8,18 +8,26 @@
 class PlaybookValidator(BaseValidator):
     """PlaybookValidator is designed to validate the correctness of the file structure we enter to content repo."""
 
-    def is_valid_playbook(self, is_new_playbook: bool = True, validate_rn: bool = True) -> bool:
+    def is_valid_playbook(self, is_new_playbook: bool = True,
+                          validate_rn: bool = True,
+                          is_test_playbook: bool = False) -> bool:
         """Check whether the playbook is valid or not.
 
          Args:
+            is_test_playbook (bool): whether the playbook is a test playbook
             is_new_playbook (bool): whether the playbook is new or modified
             validate_rn (bool):  whether we need to validate release notes or not
 
         Returns:
             bool. Whether the playbook is valid or not
         """
-
-        if is_new_playbook:
+        if is_test_playbook:
```

Comment:
why do we need special case for test playbooks?

Reply:
Not sure about the specifics but all validators simply skip test playbooks for some reason, If I recall correctly @yaakovi told me this is a check we can add instead of skipping them.

Reply:
![image](https://user-images.githubusercontent.com/41257953/79755802-5667ee80-8322-11ea-8abd-d7102b1d3788.png)

Reply:
spoke with @yaakovi, sorry for the confusion but no real need to have special case for test playbooks, id prefer we stay consistent as there is no real reason not to run the same validations